third_party/Environment/IsAppEncrypted.m

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Google LLC
```

Comment:
I think the Google License should now be removed and the 3P license moved to the top of the file.

Reply:
Good point. Done in the IsAppEncyrpted.[hm]. I adjusted the copyright notice to have max 80 chars on each line.